__tests__/generate-nearby.test.js

```diff
@@ -0,0 +1,54 @@
+const faker = require('faker');
+const count = 5;
+let generateTest = (total) => {
```

Comment:
Why are you defining and testing genreateTest in this file?  We want to test the function that actually makes the data in generate-nearby.js